lib/sass/script/functions.rb

```diff
@@ -211,8 +216,8 @@ module Sass::Script
   # \{#map_values map-values($map)}
   # : Returns a list of all values in a map.
   #
-  # \{#map_has_key map-has-key($map, $key)}
-  # : Returns whether a map has a value associated with a given key.
+  # \{#map_has_key map-has-key($map, $keys...)}
+  # : Returns whether a map has a value associated with a given key or keys.
```

Comment:
"key or keys" makes this sound like it checks multiple keys in the base map. Maybe "Returns whether a map has a (potentially nested) value associated with the given key(s)."